wild/tests/sources/init-order.c

```diff
@@ -0,0 +1,35 @@
+//#Object:runtime.c
```

Comment:
It might help to add `//#Object:init.c`, then `#include "init.h"` and in `_start` call `call_init_functions();`. See `init_test.c` for an example. This will cause GNU ld to see that `__start_init_array` and `__end_init_array` are referenced so it will emit those symbols. It'll also mean that the init functions will get run when the program runs.